src/AspNetCore/Src/inprocessapplication.cxx

```diff
@@ -56,10 +58,36 @@ EXTERN_C __MIDL_DECLSPEC_DLLEXPORT VOID http_set_response_status_code(
 EXTERN_C __MIDL_DECLSPEC_DLLEXPORT
 HRESULT
 http_post_completion(
-    _In_ IHttpContext* pHttpContext
+    _In_ IHttpContext* pHttpContext,
+    DWORD cbBytes
 )
 {
-    return pHttpContext->PostCompletion(0);
+    return pHttpContext->PostCompletion(cbBytes);
+}
+
+EXTERN_C __MIDL_DECLSPEC_DLLEXPORT
+HRESULT
+http_set_managed_context(
+    _In_ IHttpContext* pHttpContext,
+    _In_ PVOID pvManagedContext
+)
+{
+    HRESULT hr;
+    IN_PROCESS_STORED_CONTEXT* pInProcessStoredContext = new IN_PROCESS_STORED_CONTEXT(pHttpContext, pvManagedContext);
+    if (pInProcessStoredContext == NULL)
+    {
+        return E_OUTOFMEMORY;
+    }
+
+    IHttpModuleContextContainer* pModuleContextContainer = pHttpContext->GetModuleContextContainer();
```

Comment:
no need to do null check for this contextcontainer. if it is null, IIS is in broken state